pkg/modelzooserver/modelzooserver_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 The SQLFlow Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package modelzooserver
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net"
+	"testing"
+	"time"
+
+	"google.golang.org/grpc"
+	pb "sqlflow.org/sqlflow/pkg/proto"
+)
+
+func startServer() {
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", 50055))
+	if err != nil {
+		log.Fatalf("failed to listen: %v", err)
+	}
+	grpcServer := grpc.NewServer()
+	pb.RegisterModelZooServerServer(grpcServer, &modelZooServer{})
+
+	grpcServer.Serve(lis)
+}
+
+func serverIsReady(addr string, timeout time.Duration) bool {
+	conn, err := net.DialTimeout("tcp", addr, timeout)
```

Comment:
Shall we put these kind of func to a dedicated package？I noticed we have so many copy of them.

Reply:
Thanks for reminding, I'll refactor this in another PR.